### PR TITLE
Update tfjs-core version / Fix explicit padding bug of AVGPOOL

### LINKED ISSUE #380: [WebGL] The outputs don't match the expected values for AVERAGE_POOL_2D operation with WebGL backend
**Test Env:**
**Chromium** nightly build 70.0.3503.0 (**6540a81**) 
 Platform: Mac/Linux/Windows/Android

**Expected Result:**
Tests should pass.

**Actual Result:**
The outputs don't match the expected values for AVERAGE_POOL_2D operation with WebGL backend:
check result for Avg pool float example/2:
image

check result for Avg pool float example/4:
image

**How to Reproduce:**
1. Launch chromium and disable WebML flag
2. Visit https://brucedai.github.io/nt/test/avg_pool_2d.html?backend=webgl
    - check result for Avg pool float example/2
    - check result for Avg pool float example/4

_They passed before with webgl2 backend._

### PR DESCRIPTION
**Pull request by GreyZzzzzzXh**
Fix #380

### CODE REVIEW

File: src/nn/webgl/WebGLModel.js [hunk 1]
```diff
@@ -263,18 +287,32 @@ export default class WebGLModel {
...
+            }            
           } else {
-            output.assign(activation(
-                input.pad([[0, 0], [paddingTop, paddingBottom],
-                           [paddingLeft, paddingRight], [0, 0]])
-                     .maxPool([filterH, filterW],
-                              [strideH, strideW], 'valid')));
+            if (op === OperationCode.AVERAGE_POOL_2D) {
+              throw new Error(
+                  'AVERAGE_POOL_2D with unequal padding is not supported');
```
Comment (GreyZzzzzzXh) [inline src/nn/webgl/WebGLModel.js]: https://js.tensorflow.org/api/0.14.1/#avgPool

**Comment by GreyZzzzzzXh** [comment]
PR tensorflow/tfjs-core#1421 is merged.
@huningxin , PTAL. Thanks.

**Comment by huningxin** [comment]
It's fantastic! Thanks for the PR!

**Comment by huningxin** [comment]
Hi @BruceDai , please help verify the accuracy improvement on Android devices with webgl 2.0 support.

**Comment by BruceDai** [comment]
#380 has been verified with this PR, thanks.

**Pull request merged**